app/models/edition/lead_image.rb

```diff
@@ -34,8 +34,6 @@ def lead_image_caption
   end
 
   def lead_image_has_all_assets?
```

Comment:
this function seems redundant now. If we delete it, or delegate it, we can then delete the related tests as well

Reply:
But the method is used in case study presenter and news article presenter, it has to be defined within the concern. There are no filtering tests at the presenter level for lead image assets missing. It would have been too repetitive to sprinkle those tests everywhere that includes the lead image concern. So these little tests cover that a bit, albeit shallowly.